### PR TITLE
ROC-3163: Reorder, refactor and change content to eligibility

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-3163

### Change description
Updated the order of eligibility to reflect the new story flow:

1. Find out if you can use this service
2. Total amount you're claiming
3. Do you need help with court fees
4. Do you live in the UK         
5. Is the defendant's address in England and Wales   
6. Are you 18 or over
7. Who are you making the claim for?
8. Is this claim against more than one person
9. Are you claiming against a gov department 
10. Is this claim for a tenancy deposit
11. You can use this service 

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [x] New pages included in a11y tests
- [x] Task list and task completeness checks updated
- [] Check and send page updated
- [x] Routes, page content and page flows of new features are toggled 
- [x] UI changes look good on mobile

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/claim/form/models/eligibility/validDefendant.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export class ValidDefendant {
```
Comment (geek-soft) [inline src/main/features/claim/form/models/eligibility/validDefendant.ts]: I do not think class name correctly reflects the purpose of that class. Options more reflect who is making a claim not who is the defendant on the claim e.g. `REPRESENTATIVE_CLAIM`. Shall we name the class `ClaimType` or something that indicates this is about the claimant not defendant?

File: src/test/features/claim/routes/eligibility/not-eligible.ts [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-
-import { Paths as ClaimPaths } from 'claim/paths'
-
-import { app } from '../../../../../main/app'
-
-import * as idamServiceMock from '../../../../http-mocks/idam'
-
-const cookieName: string = config.get<string>('session.cookieName')
-
-describe('Claim eligibility: index page', () => {
```
Comment (geek-soft) [inline src/test/features/claim/routes/eligibility/not-eligible.ts]: Why is this file removed? I would say it. should have got fixed not removed.

Comment (jacmurphy) [reply src/test/features/claim/routes/eligibility/not-eligible.ts]: The file did nothing and was just a copy paste error

File: src/test/features/claim/routes/eligibility/index.ts [hunk 3]
```diff
@@ -13,6 +13,8 @@ import { app } from '../../../../../main/app'
 import * as idamServiceMock from '../../../../http-mocks/idam'
 
 const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = ClaimPaths.eligibilityStartPage.uri
```
Comment (geek-soft) [inline src/test/features/claim/routes/eligibility/index.ts]: Can you please update line 23 to use `pagePath` instead of `ClaimPaths.startPage.uri`?

File: src/test/features/claim/routes/eligibility/not-eligible.ts (hunk 2 above)
Comment (geek-soft) [reply src/test/features/claim/routes/eligibility/not-eligible.ts]: True, but it was supposed to cover `routes/eligibility/not-eligible.ts`. We should fix issues not remove tests.

File: src/main/features/claim/form/models/eligibility/validDefendant.ts (hunk 1 above)
Comment (jacmurphy) [reply src/main/features/claim/form/models/eligibility/validDefendant.ts]: After reading this i messaged you in slack and talked about it. To summarise the conversation, im changing the name to ClaimType as you suggested

File: src/test/features/claim/routes/eligibility/not-eligible.ts (hunk 2 above)
Comment (jacmurphy) [reply src/test/features/claim/routes/eligibility/not-eligible.ts]: You're right, my mistake. adding now

File: src/test/features/claim/routes/eligibility/index.ts (hunk 3 above)
Comment (jacmurphy) [reply src/test/features/claim/routes/eligibility/index.ts]: 👍
